.github/workflows/ci.yaml

```diff
@@ -20,3 +20,10 @@ jobs:
       env:
         PYTHONPATH: ${{github.workspace}}/ros_buildfarm
       run: ${{github.workspace}}/ros_buildfarm/scripts/misc/validate_config_index.py file://${{github.workspace}}/index.yaml
+  yamllint:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Validate YAML style
+      run: |
+        yamllint -d "{extends: default, rules: {indentation: {indent-sequences: false}, line-length: disable}}" .
```

Comment:
What do you think about setting a local .yamllint config rather than using this cli arg so that `yamllint` can be easily run locally as well as in the workflow?

Reply:
Sounds great. Done.